ci/testdata/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM golang:1.19-alpine
 # Install bash and git (and build-base to get gcc)
 # (this is required when building FROM: golang:alpine)
 RUN apk update && apk add --no-cache git bash build-base
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 RUN wget -O- -nv https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.41.1
```

Comment:
Just discovered: Turns out we don't need to install `golangci-lint` using this `wget` thing...

We can simply do:
```sh
RUN apk update && apk add --no-cache git=~2.47 bash=~5.2.37 build-base=~0.5 golangci-lint=~1.61
```
Then we don't need the `pipefail` thing anymore.